mlir/unittests/Dialect/QCO/Transforms/Optimization/test_qco_hadamard_lifting.cpp

```
@@ -0,0 +1,516 @@
+#include <llvm/ADT/SmallVector.h>
+#include <mlir/Dialect/Arith/IR/Arith.h>
+#include <mlir/Dialect/Func/IR/FuncOps.h>
+#include <mlir/IR/BuiltinOps.h>
+#include <mlir/IR/DialectRegistry.h>
+#include <mlir/IR/OwningOpRef.h>
+#include <mlir/IR/Value.h>
+#include <mlir/Pass/PassManager.h>
+#include <mlir/Support/LogicalResult.h>
+#include <mlir/Transforms/Passes.h>
+
+#include <cassert>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)